test/regridder_tests.jl

```diff
@@ -144,179 +142,162 @@ for FT in (Float32, Float64)
     # TempestRemap is not built on Windows because of NetCDF support limitations
     if !Sys.iswindows()
         @testset "test write_to_hdf5 and read_from_hdf5" begin
-            # Set up testing directory
-            ispath(REGRID_DIR) && rm(REGRID_DIR; recursive = true, force = true)
-            mkpath(REGRID_DIR)
-
-            hd_outfile_root = "hdf5_out_test"
-            tx = Dates.DateTime(1979, 01, 01, 01, 00, 00)
-            test_space = TestHelper.create_space(FT)
-            input_field = CC.Fields.ones(test_space)
-            varname = "testdata"
-
-            Regridder.write_to_hdf5(REGRID_DIR, hd_outfile_root, tx, input_field, varname, comms_ctx)
-
-            output_field = Regridder.read_from_hdf5(REGRID_DIR, hd_outfile_root, tx, varname, comms_ctx)
-            @test parent(input_field) == parent(output_field)
-
-            # Delete testing directory and files
-            rm(REGRID_DIR; recursive = true, force = true)
+            # Set up regrid directory for this test only
+            mktempdir(".") do regrid_dir
```

Comment:
```suggestion
            mktempdir(pwd()) do regrid_dir
```
maybe this is more robust for windows. I don't know